packages/terra-tag/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-tag",
+  "main": "lib/Tag.js",
+  "version": "0.0.0",
+  "description": "The tag component is used for tagging and provides users a way to trigger actions in the UI. Each tag should have text. It can optionally display an icon along with the text.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-tag",
+    "Tag",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "terra-props-table": "^1.10.1"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^2.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-base": "^2.0.0"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rimraf lib",
+    "compile:build": "babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "props-table": "props-table ./src/Tag.jsx --out-dir ./docs/props-table",
```

Comment:
Remove `props-table` script. This change occured in #1301

Reply:
Updated it [here](https://github.com/cerner/terra-core/pull/1306/commits/60823599eb0b3581ec58dcd340f9b88de2b87714).